features/old/cms/advanced/multi-tenant/authorizations/groups_and_sections.feature

```diff
@@ -1,3 +1,4 @@
+@no-txn
```

Comment:
This should not be needed. It was not before upgrade to Cucumber 7. Also it is not needed for MySQL

I think the issue lies on some DB feature.

Reply:
@hallelujah , before #2579 test ran in `truncation` mode. If you check references in description, especially  cucumber/cucumber-rails#166 people report that drivers are not thread safe and strange effects may appear.

This PR makes switching between `truncation` and `transaction` mode work. Before it, only `@javascript` tagged scenarios ran in `truncation` mode.

I don't know what kind of database feature may affect this scenario. Please provide any suggestions. It might be something specific about how Postgresql driver works because some people reported one thread doing a query and another thread receiving the result.